database/fruits.ts

```diff
@@ -11,3 +11,11 @@ export const fruits: Fruit[] = [
   { id: 4, name: 'Mango', icon: '🥭' },
   { id: 5, name: 'Avocado', icon: '🥑' },
 ];
+
+export function getFruitById(id?: number) {
```

Comment:
this should always require an `id`, no?

```suggestion
export function getFruitById(id: number) {
```

Reply:
c17f227